components/InfoPane/InfoPane.tsx

```diff
@@ -27,10 +27,17 @@ export default function InfoPane(props: HTMLProps<HTMLDivElement>) {
             package.json file
           </QueryLink>
         </li>
-        <li>package.json pastes (copied as text or file)</li>
       </ul>
 
-      <FileUploadControl />
+      <p>It also accepts package.json:</p>
+      <ul>
+        <li>Drag and drop a file anywhere on the page</li>
+        <li>Paste package.json (copied as text)</li>
+        <li>Paste package.json (copied as file)</li>
+        <li>
+          <FilePicker label="Choose file" /> from your computer
+        </li>
+      </ul>
```

Comment:
The drop area is gone and replaced by extended instructions. I'm open to make further style improvements. Ideally these would be part of a dedicated "Documentation" section but I don't want to move the simple text-only examples too far away from the field.

Reply:
Works for me!